### ISSUE TITLE
Command line flag for single instance

### ISSUE THREAD

**Issue by oskretc** [fman-community/fman#464]
Maybe there is something already but couldn't find it.

Is there a way to use fman from cli and open the directories in a single instance, e.g. if not running open new fman instance, if running, change that instance to the directory specified in the argument(s)

**Comment by mherrmann** [comment]
Funny, I was just working on single-instance mode in the past few days. It isn't yet implemented, and actually not easy to do correctly. Which OS did you say you were on?

**Comment by oskretc** [comment]
Win 7 at work and win 10 at home

**Comment by puffitos** [comment]
Did you manage to get around to implementing this? Or does someone have a workaround/ an idea on how to enforce a single instance on a Linux system?

**Comment by mherrmann** [comment]
@puffitos couldn't you just write a relatively simple bash script?

**Comment by puffitos** [comment]
Ah ok, and then it would wrap the fman binary. My aim is to use only fman as the default file manager to handle "file-manager-related" MIME Types. I'll try that, thanks :+1:

**Comment by amarbota** [comment]
> @puffitos couldn't you just write a relatively simple bash script?

How would the bash/batch script look like? I am not quite sure if that would be achievable with a bash script still you need to point to the fman.exe and as soon you do it it opens a new instance of fman. 

@mherrmann I used TotalCommander, currently buying FreeCommander licenses but fman is really great that I would immediately jump over if I could pass the path to a folder that I want to open in fman. I use cli a lot and also in my case zim-desktop-wiki so that I work a lot with different folders so I need a quick way to open the folder I currently work in. 

Something like this would be great:
https://freecommander.com/fchelpxe/en/Commandlineparameters.html

Do you think it is possible to add the functionality with a custom plugin?

**Comment by mherrmann** [comment]
@amarbota yes, I think you could write a plugin that listens on a port for commands that change the left or right pane. Then send commands to this port from the bash/batch script. Maybe it's not _that_ simple. But it's certainly possible.

**Comment by amarbota** [comment]
> @amarbota yes, I think you could write a plugin that listens on a port for commands that change the left or right pane. Then send commands to this port from the bash/batch script. Maybe it's not _that_ simple. But it's certainly possible.

Hi @mherrmann 

Can you  please point tp the api or port which fman is using. I will give it a try.

Danke!

**Comment by mherrmann** [comment]
fman is not using a port. You would need to open it from the plugin. The API can be googled with "fman api".
